README.md

```diff
@@ -1,103 +1,485 @@
 
-Lorem ipsum dolor sit amet. Ut velit fugit et porro voluptas quia sequi quo
-libero autem qui similique placeat eum velit autem aut repellendus quia. Et
-Quis magni ut fugit obcaecati in expedita fugiat est iste rerum qui ipsam
-ducimus et quaerat maxime sit eaque minus. Est molestias voluptatem et nostrum
-recusandae qui incidunt Quis 33 ipsum perferendis sed similique architecto.
+**NOTE: Remember to run `pipenv install` to install the dependencies and
+`pipenv shell` to enter your virtual environment before running your code.**
+
+```bash
+pipenv install
+pipenv shell
+```
+
+The `lib` folder contains starter code for the `Department` and `Employee`
+classes. Both classes include ORM methods for mapping the individual class to a
+corresponding table in the database:
+
+| Python Class | Relational Database Table |
+| ------------ | ------------------------- |
+| Department   | departments               |
+| Employee     | employees                 |
+
+The one-to-many relationship between `Department` and `Employee` is **not**
+implemented in the starter code. In this lesson, we will:
+
+- Update the `Employee` class to add a new attribute to model the one-to-many
+  relationship.
+- Update the `Employee` class methods to store and manage the relationship in
+  the "employees" database.
+- Update the `Department` class to add a new method to **compute** (not store) a
+  list of associated `Employee` class instances.
+
+---
+
+## Mapping a one-to-many relationship
+
+The concept of **single source of truth** states we should design our software
+and data to avoid redundancy. In terms of relational database design, we want to
+store the relationship between two entities in just one place within the
+database. We do this to avoid inconsistencies that often arise when redundant
+data is not updated in a consistent manner.
+
+When we have a relationship between two classes, we need to figure out where to
+store the relationship. For the one-to-many relationship, the class (or entity)
+on the "many" side is the owner of the relationship and is responsible for
+storing the relationship. Why? Beside each object on the "many" side (i.e.
+Employee) is related to just one entity on the "one" side (i.e. Department),
+thus it only needs to store a single value to maintain the relationship.
+
+![owning side](https://curriculum-content.s3.amazonaws.com/7134/python-p3-v2-orm/owning_side.png)
+
+`Employee` is on the many side of the relationship, thus it **owns** the
+relationship and is responsible for managing the relationship by storing a
+single piece of data about the associated `Department`.
+
+- The "employees" table will contain a column to store a foreign key reference
+  to the "departments" table.
+
+`Department` is on the "one" side of the relationship, thus it should not
+**store** a list of the associated `Employee` instances. Instead, a list of the
+many `Employee` objects associated with the will be **computed** by looking at
```

Comment:
Fix: typo

"...many `Employee` objects associated with the will be..."

Reply:
fixed